### DISCUSSION TITLE
connecting drf1605h module to raspberry pi4

### DISCUSSION THREAD

**Discussion by mojijafari** [Koenkk/zigbee2mqtt#5363] [General]
Dear Koenkk,
Thank you very much for your kind support, is there any chance to connect drf1605h to raspberry pi 4?
I saw the datasheet but still can't connect it. if it is possible, what is the serial?

Thank you in advance

**Comment by Koenkk** [comment]
I don't know about this device but perhaps somebody else can help

**Reply by mojijafari** [reply]
thank you very much for your kind answer. is there any chance to send you the datasheet. In my area, we don't have any other module. Or if some one else can help me, could you link me to?
